docs/virtual_machines/interfaces_and_networks.md

```diff
@@ -595,6 +595,48 @@ has no DHCP server at all. Therefore, the VM won't have the search domains infor
 reaching a destination using its FQDN is not possible.
 Tracking issue - https://github.com/kubevirt/kubevirt/issues/7184
 
+### passt
+
+`passt` is a new approach for user-mode network that can be used as a simple replacement for Slirp.
+
+```yaml
+kind: VM
+spec:
+  domain:
+    devices:
+      interfaces:
+        - name: red
+          passt: {} # connect using passt mode
+  networks:
+  - name: red
+    pod: {}
+```
+
+#### Requirements:
+1. To get better performance the node should be configured with:
+```
+sysctl -w net.core.rmem_max = 33554432
+sysctl -w net.core.wmem_max = 33554432
+```
+2. To run a multiple passt VMs with no explicit ports, the node's `fs.file_max` should be increased
```

Comment:
```suggestion
2. To run multiple passt VMs with no explicit ports, the node's `fs.file_max` should be increased
```

What if user fails to do this? Is it worth adding a "debugging" section that would help users to easily identify that this must be increased?
Or maybe just a note saying tha if `this` happens, it's because of `that`

Reply:
If the user doesn't specify it, he will fail running the VM.

Reply:
I don't want to specify here the exact error the user will get since it is planed to add a validation webhook making sure the are enough free FDs with an explicit error.

Reply:
I see, that sound better than my suggestion :)